src/F0_output.py

```diff
@@ -250,15 +250,16 @@ def convert(o):
     # todo this actually drops the date time index, which could be interesting
     elif isinstance(o, pd.DatetimeIndex):
         answer = "date_range"
+        answer = o.to_frame().to_json(orient="split")
```

Comment:
```suggestion
        answer = o.to_frame().to_json(orient="split")
```
Was there a reason you kept ´answer = "date_range"?